app/models/firm.rb

```diff
@@ -173,6 +174,10 @@ def primary_advice_method
     infer_primary_advice_method
   end
 
+  def main_office
+    offices.first
```

Comment:
Umm I'm confused. Does this help? ...

https://github.com/moneyadviceservice/rad/blob/multiple_offices/app/views/self_service/offices/_offices_table.html.erb#L32

and

![screen shot 2015-08-14 at 14 14 07](https://cloud.githubusercontent.com/assets/306583/9277051/a30f0fac-429f-11e5-9121-8fff949e868a.png)